### ISSUE TITLE
search or paginating without filters - ExpressionAttributeNames error

### ISSUE THREAD

**Issue by jpmtrabbold** [benhutchins/dyngoose#478]
Hey guys. First of all - what amazing work you put into this library, I cannot be more grateful.

I'm using version 2.16.4, and whenever I do a search with no filters (whether to paginate or not), the following error is thrown:

"ExpressionAttributeNames can only be specified when using expressions"

```typescript
        // THIS WORKS
        const results = await Farm.search().filter('name').not().eq('3242342342').exec()

        // THIS DOESN'T
        const results2 = await Farm.search().exec() // ExpressionAttributeNames can only be specified when using expressions

        // THIS DOESN'T
        const results3 = await Farm.search().all() // ExpressionAttributeNames can only be specified when using expressions
```

My workaround at the moment is to leave a nonsensical filter there, just so the query works - but hoping for a solution so I can share this library with my coworkers unashamedly :) 

thank you!

**Comment by benhutchins** [comment]
Thank you for reporting this @jpmtrabbold. This should be fixed as part of [2.16.5](https://github.com/benhutchins/dyngoose/releases/tag/v2.16.5)

**Comment by jpmtrabbold** [comment]
@benhutchins you're quick man, thanks!
